tests/plugin/test_plugin.py

```
@@ -57,6 +58,91 @@ def test_create_user(self, browser_layers):
+    ):
+        """authenticateCredentials should auto-create the Plone user on first successful JWT auth."""
+        payload = {
+            "grant_type": "password",
+            "client_id": "keycloak-idp",
+            "client_secret": "12345678910",
+            "username": "kimug",
+            "password": "kimug",
+            "scope": ["openid"],
+        }
+        headers = {"Content-Type": "application/x-www-form-urlencoded"}
+        response = requests.post(
```

Ruff (0.15.14)

[error] Probable use of `requests` call without timeout

(S113)